source/grid/tria.cc

```diff
@@ -12771,6 +12771,39 @@ Triangulation<dim, spacedim>::update_periodic_face_map()
 }
 
 
+template <int dim, int spacedim>
```

Comment:
Can you use 3 empty lines between the functions here and below?

Reply:
I can do that. But 2 lines are the norm in tria.cc, which is why I did not use 3 lines.

Reply:
I saw both in the file with maybe a slight tendency to 2 lines, but 3 lines are also quite common; the reason why I mentioned it is that the place where you inserted the code used to have 3 lines of empty space :wink:

Reply:
Done!